make/ocm.mk

```diff
@@ -0,0 +1,48 @@
+OCM_IMAGE=registry.svc.ci.openshift.org/openshift/release:intly-golang-1.12
```

Comment:
This isn;t used since you removed the docker thing i had here https://github.com/mikenairn/integreatly-operator/commit/812c00b2982b7000dd86846778c5246557edbd0e#diff-b67911656ef5d18c4ae36cb6741b7965R222

Reply:
do you want the docker thing to be added?